### ISSUE TITLE
Is file only from latest release served?

### ISSUE THREAD

**Issue by develar** [kevinphelps/electron-update-server#1]
>  Download file from the latest release in the given repo

Does it means that downside is that url always points to a latest release files? And no way to download file from previous release?

**Comment by kevinbuhmann** [comment]
Yes. Currently, only the latest release is support. It should be fairly simple to add an optional release tag name in the download routes. I can work on that when I get time. Pull requests also welcome. Thanks!

**Comment by kevinbuhmann** [comment]
Sorry, I didn't mean to close this.
